modules/bibauthorid/lib/bibauthorid_daemon.py

```diff
@@ -149,7 +171,26 @@ def _task_submit_elaborate_specific_parameter(key, value, opts, args):
         value = [token.strip() for token in value.split(',')]
         bibtask.task_set_option("last-names", value)
     elif key in ("--single-threaded",):
-        bibtask.task_set_option("single-threaded", True)
+        bibtask.task_set_option("single_threaded", True)
+    elif key in ("--force-identifier-consistency",):
+        bibtask.task_set_option("force_identifier_consistency", True)
+    elif key in ("--check-db",):
+        bibtask.task_set_option("check_db", True)
+    elif key in ("--do-not-modify-records",):
+        bibtask.task_set_option("dry_run", True)
+    elif key in ("--packet-size"):
+        if value.count("="):
+            value = value[1:]
+        value = int(value)
+        bibtask.task_set_option("packet_size", value)
+    elif key in ("--do-not-modify-hepnames",):
+        bibtask.task_set_option("dry_hepnames_run", True)
+    elif key in ("--open-tickets",):
+        bibtask.task_set_option("open_tickets", True)
+    elif key in ("--queue",):
+        if value.count("="):
+            value = value[1:]
+        bibtask.task_set_option("queue", value)
```

Comment:
It's not an issue with the help message... it's how `getopt` works (bibtasks CLI is based on getopt):
https://docs.python.org/2/library/getopt.html#getopt.getopt

you have to have a trailing `=` to parameter added in `specific_params` if an argument is expected (as it is the case for `--queue=` and I now notice `--packet-size=`. With a trailing `=` it will be getopt to correctly pass you the value (no stripping of `=` is then necessary).